src/vehicle_use.cpp

```diff
@@ -359,6 +361,19 @@ void vehicle::set_electronics_menu_options( std::vector<uilist_entry> &options,
             refresh();
         } );
     }
+
+    if( has_part( "ARCADE" ) )
+    {
```

Comment:
```suggestion
    if( has_part( "ARCADE" ) ) {
```
Small astyle correction (at least it compiles now) :P

Reply:
It compiles now but all appliances are now standing lamps when I tried constructing a grid with hammerspace to test it.

Reply:
I tried it without debug hammerspace and it seems to work fine:

![app](https://user-images.githubusercontent.com/12537966/156698633-70736a18-9da8-42c8-b434-eddae2b2d7d3.png)

I think that hammerspace bug is unrelated to this PR, since it happens on the master builds as well.